arwen/elrondapi/eei_helpers.go

```diff
@@ -61,7 +61,8 @@ func getESDTRoles(data_buffer []byte) int64 {
 	return result
 }
 
-func validateToken(tokenID []byte) int32 {
+// ValidateToken - validates the token ID
+func ValidateToken(tokenID []byte) int32 {
```

Comment:
I think we should return bool here and only convert to int32 when returning the value to the contract.